scripts/helper/executeAll.js

```diff
@@ -1,30 +1,77 @@
-const proc = require('child_process');
+const spawn = require('child_process').spawn;
 const path = require('path');
 
-let errors = false;
+let error = false;
 
 /**
- * Execute the given command for the given project
- * relative from the root-folder.
- * if an error occurs it set true to the global error-variable
+ * check if the error-message is as npm-warning
+ * to prevent build-break
  *
- * @param  {string} command shell-command
- * @param  {string} project project-name
- * @return {void}
+ * @param  {string}   msg  error-message
+ * @return {boolean}       is npm warning
  */
-function executeCommand(command, project) {
-  console.log(`start ${command} for ${project}`);
+function isNPMWarning(msg) {
+  return msg.indexOf('npm') !== -1
```

Comment:
doens´t work because the key´s are part of a message and not the complete message.

following would work:
['npm', 'WARN', 'deprecated', ...].filter(el => msg.indexOf(el) !== -1).length